platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/VirtualKey.java

```diff
@@ -54,6 +54,7 @@ public interface VirtualKey extends SelfSerializable {
      * @throws IOException
      * 		If there was a problem writing this classes data into the ByteBuffer
      */
+    // TODO: serialize(WriteableSequentialData)
```

Comment:
Don't forget either remove `// TODO:` or implement it

Reply:
This comment is removed in the latest version of the PR, and both `serialize()` and `deserialize()` methods are now marked as deprecated. At some point we'll need further cleanup of these interfaces, so all serialization code is removed from keys/values and left in key/value serializers only.